message_definitions/v1.0/common.xml

```diff
@@ -3495,7 +3495,7 @@
       <field type="int32_t" name="baseline_a_mm" units="mm">Current baseline in ECEF x or NED north component in mm.</field>
       <field type="int32_t" name="baseline_b_mm" units="mm">Current baseline in ECEF y or NED east component in mm.</field>
       <field type="int32_t" name="baseline_c_mm" units="mm">Current baseline in ECEF z or NED down component in mm.</field>
-      <field type="uint32_t" name="accuracy">Current estimate of baseline accuracy.</field>
+      <field type="uint32_t" name="accuracy" units="c%">Current estimate of baseline accuracy</field>
```

Comment:
This makes no sense for "normal" users, Like I said it makes perfect sense to Emlid users. This is the reason for my "agressive" patch split. I know some parts are controversial, like this one, and I move them away to separated PRs. This way the non-controversial parts have a chance of getting in.

This specific PR makes sence to Emlid users, and should get us thinking about solutions to meet the need of all GPSs, without stalling the merge of the other PRs.